homeassistant/components/tahoma/__init__.py

```diff
@@ -42,26 +45,33 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry):
 
     username = entry.data.get(CONF_USERNAME)
     password = entry.data.get(CONF_PASSWORD)
+    hub = entry.data.get(CONF_HUB, DEFAULT_HUB)
+    endpoint = SUPPORTED_ENDPOINTS[hub]
 
     session = aiohttp_client.async_create_clientsession(hass)
-    client = TahomaClient(username, password, session=session)
+    client = TahomaClient(
+        username,
+        password,
+        session=session,
+        api_url=endpoint,
+    )
 
     try:
         await client.login()
         devices = await client.get_devices()
         scenarios = await client.get_scenarios()
         places = await client.get_places()
     except BadCredentialsException:
-        _LOGGER.error("Invalid authentication.")
+        _LOGGER.error("Invalid authentication")
```

Comment:
Not related to this PR, but apparently the pylint rules did change after last rebase to `rework_tahoma`.